README.md

```diff
@@ -1 +0,0 @@
-# Past-Project-Review
```

Comment:
You need to update your read me file with details of your project.

Reply:
Nice review point, but why should the student make such a change? Kindly provide an explanation, a "why", for each requested change to help the student know exactly why he is to make such changes. :wink: